build.gradle

```diff
@@ -1,18 +1,15 @@
 plugins {
 	id 'maven-publish'
-	alias libs.plugins.quilt.loom
+	id "com.modrinth.minotaur" version "2.4.5"
```

Comment:
Since we fixed minotaur in libs.versions.toml, we *should* be able to go back to `alias(libs.plugins.minotaur)` here without errors

Reply:
It appears the github action didn't spit out any errors after making this change.